Makefile

```diff
@@ -5,7 +5,8 @@ SHELL = bash
 RHOAS_VERSION ?= "dev"
 REPOSITORY_OWNER ?= "redhat-developer"
 REPOSITORY_NAME ?= "app-services-cli"
-TERMS_SPEC_URL ?= "https://console.redhat.com/apps/application-services/terms-conditions-spec.json"
+## TODO replace with the real value
+CONSTANTS_URL ?= "https://raw.githubusercontent.com/redhat-developer/app-services-ui/main/static/configs/service-constants.json"
```

Comment:
This will need to be changed to production next year once we get this changes in prod